extensions/dicom-seg/src/components/SegmentationPanel.css

```diff
@@ -0,0 +1,176 @@
+.labelmap-container {
```

Comment:
Only concern here is collision with other extensions. It may be worth considering an extension vendor prefix, shifting these to a shared library like `@ohif/ui`, or something that produces component-scoped styles that don't bleed out.

Reply:
Done. Like commented above, since we will not be using those components outside this extension for now, I just scoped all styles within its corresponding extension component main style, this allows us to move them easily in the future.